cypress/DATA_CY_SELECTORS.md

```
@@ -0,0 +1,203 @@
+# Convention des Sélecteurs Data-Cy
+
+Ce document décrit les conventions utilisées pour les sélecteurs `data-cy` dans les tests E2E Cypress de ce projet.
+
+## Philosophie
+
+Les sélecteurs `data-cy` offrent plusieurs avantages par rapport aux sélecteurs CSS classiques :
+- **Stabilité** : Indépendants des changements de styles et classes CSS
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)